### DISCUSSION TITLE
Οδηγίες εγκατάστασης funtoo Linux σε virtual machine

### DISCUSSION THREAD

**Discussion by NickLitharis** [courses-ionio/sw#1178] [Show and tell]
Καλησπέρα σε όλους, ⁣:smiley: 

Με τον [οργανισμό](https://github.com/PaneksypnesDiepafes) μου φτιάξαμε ένα σύντομο και περιεκτικό [οδηγό](https://github.com/PaneksypnesDiepafes/cookbook/blob/main/funtoo-installation.md) για την εγκατάσταση του funtoo Linux σε εικονική μηχανή και πιο συγκεκριμένα στο Virtual Box.

**Comment by NickLitharis** [comment]
@epidrome σας κάνω mention για να κάνετε pin το guide

**Reply by epidrome** [reply]
@NickLitharis ευχαριστούμε! Για όσους τον ακολουθήσουν θα πρέπει να θυμούνται ότι περισσότερο από οδηγίες-οδηγούς θα πρέπει να τεκμηριώσουν τις επιλογές στο λογισμικό τους αναφορικά με αρχιτεκτονική και υλικά, όπως περιγράφονται θεωρητικά στις βίντεο διαλέξεις

**Reply by p20sami** [reply]
@NickLitharis Μια εναλλακτική στο βήμα 4 του Exit Environment and Reboot που δοκίμασα στην εγκατάσταση που έκανα είναι ότι στις ρυθμίσεις της μηχανής>Σύστημα έβαλα την ακολουθία του boot να είναι Σκληρός δίσκος και μετά οπτικός δίσκος, οπότε τη πρώτη φορά που έκανα boot με έβγαλε κανονικά στο iso και όταν έφτασα στο σημείο του reboot έκανα το reboot με την εντολή και με έβγαλε στα funtoo που έγιναν εγκατάσταση οπότε αποφεύγεται έτσι το να τερματίσουμε το vm και να αφαιρέσουμε το livecd. Οπότε αν θέλετε να το έχετε υπόψιν υπάρχει και αυτή η εναλλακτική. Επίσης επειδή είδα οτι αναφέρατε προβλήματα με την εγκατάσταση UEFI/GPT, προσωπικά το δοκίμασα αφού τσέκαρα το Ενεργοποίηση του EFI, ακολούθησα τις εντολές του [funtoo guide](https://www.funtoo.org/Install/Download_LiveCD) και δούλεψε κανονικά

**Reply by NickLitharis** [reply]
Συνάδελφε ευχαριστούμε για το σχόλιο😊, θα θέλαμε να συμπεριλάβουμε την προσθήκη σου στον οδηγό μας μέσω ενός pull request ώστε να είναι πιο ολοκληρωμένος ή ακόμα να κάνεις εσύ έναν δικό σου οδηγό εξηγώντας μια διαφορετική προσέγγιση. Όπως αναφέρουμε και στον οδηγό μας, οι οδηγίες είναι απλά ένας από τους τρόπους να το εγκαταστήσεις γι'αυτό και πρέπει πάντα να κοιτάμε τον official οδηγό.
